src/main/java/mate/academy/bookstore/config/SecurityConfig.java

```diff
@@ -0,0 +1,54 @@
+package mate.academy.bookstore.config;
+
+import static org.springframework.security.config.Customizer.withDefaults;
+
+import lombok.AllArgsConstructor;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.config.annotation.authentication.configuration.AuthenticationConfiguration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration
+@AllArgsConstructor
```

Comment:
Read about difference between annotations 
```suggestion
@RequiredArgsConstructor
```